src/components/paper-time-input.js

```diff
@@ -101,19 +110,45 @@ export class PaperTimeInput extends PolymerElement {
 
       <label hidden$="[[hideLabel]]">[[label]]</label>
       <div class="time-input-wrap">
+        <!-- Day Input -->
+        <paper-input
+          class="day-input"
+          id="day"
+          type="number"
+          value="{{day}}"
+          label="[[dayLabel]]"
+          on-change="_formatDay"
+          required=""
+          auto-validate="[[autoValidate]]"
+          prevent-invalid-input=""
+          maxlength="2"
+          max="99"
+          min="0"
+          ?no-label-float="[[!floatInputLabels]]"
+          ?always-float-label="[[alwaysFloatInputLabels]]"
+          always-float-label=""
```

Comment:
I couldn't get `?always-float-label="[[alwaysFloatInputLabels]]"` to work, what am I doing wrong?

Reply:
You're mixing Polymer and Lit. `?` at the start is a Lit thing. For Polymer just `always-float-label=[[alwaysFloatInputLabels]]`

Reply:
Thanks!